saleor/csv/utils/export.py

```diff
@@ -0,0 +1,345 @@
+import os
```

Comment:
Whoever: make sure to document this, this will save a lot of time when doing the reverse: importing from Saleor CSV.

Reply:
In the future, we want to also support the export of orders and customers. We should think about how to organize the logic in this file, so we have some abstraction for the common machinery that will be used in all cases. Maybe it's too early for this but we should keep it in mind.